Add missing dependencies to Fotran source files to allow parallel compilation

Author: Alexander Grund (TU Dresden)

diff --git a/gulp-6.0-orig/Src/Makefile b/gulp-6.0/Src/Makefile
index 17c283c..9bc5847 100755
--- a/gulp-6.0-orig/Src/Makefile
+++ b/gulp-6.0/Src/Makefile
@@ -2384,8 +2384,8 @@ realmi3.o: ../realmi3.F90 modules.o iomod.o m_trace.o m_strain.o
 realmi3d.o: ../realmi3d.F90 modules.o iomod.o m_trace.o m_strain.o
 realmi3fd.o: ../realmi3fd.F90 modules.o m_trace.o m_strain.o
 realmimc3.o: ../realmimc3.F90 modules.o m_trace.o
-realp.o: ../realp.F90 modules.o datatypes.o m_trace.o
-realpd.o: ../realpd.F90 modules.o datatypes.o m_trace.o
+realp.o: ../realp.F90 modules.o datatypes.o m_trace.o  m_strain.o
+realpd.o: ../realpd.F90 modules.o datatypes.o m_trace.o  m_strain.o
 realrecip3d3.o: ../realrecip3d3.F90 modules.o datatypes.o iomod.o m_trace.o
 realrecip3d3d.o: ../realrecip3d3d.F90 modules.o datatypes.o iomod.o m_trace.o
 realrecip3d3dV.o: ../realrecip3d3dV.F90 modules.o datatypes.o iomod.o m_trace.o
@@ -2445,10 +2445,10 @@ rfohess.o: ../rfohess.F90 modules.o m_trace.o
 rfostep.o: ../rfostep.F90 modules.o iomod.o
 rhoinv.o: ../rhoinv.F90 m_trace.o modules.o
 rhtohex.o: ../rhtohex.F90 modules.o m_trace.o
-rigidmoleculedrv.o: ../rigidmoleculedrv.F90 modules.o
+rigidmoleculedrv.o: ../rigidmoleculedrv.F90 modules.o m_strain.o
 rigidmoleculegv.o: ../rigidmoleculegv.F90 modules.o
 rigidmoleculephon.o: ../rigidmoleculephon.F90 modules.o
-rigidmoleculeprop.o: ../rigidmoleculeprop.F90 modules.o
+rigidmoleculeprop.o: ../rigidmoleculeprop.F90 modules.o m_strain.o
 rlist.o: ../rlist.F90 modules.o m_trace.o
 rpair.o: ../rpair.F90 modules.o m_trace.o
 rpair2.o: ../rpair2.F90 modules.o m_trace.o
